packages/api/src/routers/article_router.ts

```diff
@@ -61,5 +70,97 @@ export function articleRouter() {
       articleSavingRequestId: result.id,
     })
   })
+
+  router.get(
+    '/:id/:outputFormat',
+    cors<express.Request>(corsConfig),
+    async (req, res) => {
+      const id = req.params.id
+      const outputFormat = req.params.outputFormat
```

Comment:
Can we verify here that `id` and `outputFormat` are not null? Maybe verify `id` not null and `outputFormat` is in a list of items like `['mp3', 'speechMarks'].contains(outputFormat)`

Reply:
Yes, def